care/users/api/viewsets/userskill.py

```diff
@@ -18,7 +18,7 @@ class UserSkillViewSet(
 
     def get_queryset(self):
         username = self.kwargs["users_username"]
-        user = get_object_or_404(get_users(self.request.user).filter(username=username))
+        user = get_object_or_404(User, username=username)
```

Comment:
Making this change also allows anyone to update skills for any users, Update with authorization or allow only for list/retrieve calls.

Reply:
Fixed now. Added a permission class with all necessary changes. 

- [x] Only district admin and above users should be able to add or remove skills for any users

- [x] All the users should be able to view any other user's linked skills irrespective of permissions

Reply:
The user itself should be able to update their skills

Reply:
@vigneshhari Done.